client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -102,6 +104,10 @@ class CalypsoifyIframe extends Component {
 				this.props.setRoute( `${ currentRoute }/${ postId }` );
 			}
 		}
+
+		if ( 'postTrashed' === action ) {
+			page.back( this.props.postTypeTrashUrl );
```

Comment:
Instead of back, we probably should navigate to the post, page or CPT list, so dispatching a redux `NAVIGATE` event would make sense here.

Reply:
fixed on 6c988b01854ed51e141ee42ff8e4675a701d991f 😉